lib/src/browser.dart

```diff
@@ -40,6 +40,10 @@ class Browser {
 
   Version _version;
 
+  void reset() {
```

Comment:
Yeah it should probably be called in the example app for manually evaluating user agents as well as the tests. Besides that though if it isn't called in the unit tests, these tests will break because the results will have cached version results from other tests.

https://github.com/Workiva/platform_detect/pull/13/files#diff-2d160049e98e28a7166fcf00d33e3bb7R82
https://github.com/Workiva/platform_detect/pull/13/files#diff-2d160049e98e28a7166fcf00d33e3bb7R100
https://github.com/Workiva/platform_detect/pull/13/files#diff-2d160049e98e28a7166fcf00d33e3bb7R118